cmd/application-create.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/redhat-developer/ocdev/pkg/application"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+var createCmd = &cobra.Command{
+	Use:   "create",
+	Short: "create an application",
+	Run: func(cmd *cobra.Command, args []string) {
+		for _, app := range args {
```

Comment:
this looks strange, why go through all arguments? 

We should be creating just one application. `ocdev application create myApp` if more arguments provided, it should error out

Reply:
what's wrong with providing multiple applications in one command?
`ocdev application create wordpress mariadb` will create both the applications.

Reply:
I think it is confusing :-(

Reply:
fixed